### ISSUE TITLE
GPU fans stuck at near-0 rpm after locking

### ISSUE THREAD

**Issue by toydotgame** [the-cavalry/light-locker#163]
**Version:** 1.9.0-5
**OS:** 5.18.3-arch1-1
<details>
<summary>Output of <code>light-locker --debug</code>, if it's of any help:</summary>

```
[gs_debug_init] ../light-locker/src/gs-debug.c:106 (16:45:32):	 Debugging enabled
[main] ../light-locker/src/light-locker.c:138 (16:45:32):	 initializing light-locker 1.9.0
[main] ../light-locker/src/light-locker.c:139 (16:45:32):	 Platform:
gtk:        3
systemd:    yes
UPower:     yes
[main] ../light-locker/src/light-locker.c:155 (16:45:32):	 Features:
lock-after-screensaver: yes
late-locking:           yes
lock-on-suspend:        yes
lock-on-lid:            yes
settings backend:       GSETTINGS
[main] ../light-locker/src/light-locker.c:188 (16:45:32):	 lock after screensaver 5
[main] ../light-locker/src/light-locker.c:189 (16:45:32):	 late locking 1
[main] ../light-locker/src/light-locker.c:190 (16:45:32):	 lock on suspend 1
[main] ../light-locker/src/light-locker.c:191 (16:45:32):	 lock on lid 1
[main] ../light-locker/src/light-locker.c:192 (16:45:32):	 idle hint 0
[init_session_id] ../light-locker/src/gs-listener-dbus.c:2234 (16:45:32):	 Got session-id: /org/freedesktop/login1/session/_32
[find_graphical_session] ../light-locker/src/gs-listener-dbus.c:2104 (16:45:32):	 Finding a graphical session for user 1000
[init_session_id] ../light-locker/src/gs-listener-dbus.c:2245 (16:45:32):	 Got sd-session-id: /org/freedesktop/login1/session/_32
[init_seat_path] ../light-locker/src/gs-listener-dbus.c:2326 (16:45:32):	 Got seat: /org/freedesktop/DisplayManager/Seat0
[gs_listener_delay_suspend] ../light-locker/src/gs-listener-dbus.c:414 (16:45:32):	 Delay suspend
```

</details>

***

I don't know how exactly to debug this issue I've been having, but when my screen is locked (either by timeout or by a keyboard shortcut), something goes wrong with my fans' PWM values. The fans usually idle at around 500-600 rpm (as per my fan curve), but if I log back into my locked system and check, it'll be around 3-9 rpm (usually sitting on 4). This leads my idle temperature going from ~50°C to ~86°C.

I really don't know where to look for my problem specifically, but I think the steps I use to manually resolve this problem may help:
* I re-configure my fan controllers using `sudo pwmconfig`. This takes my fans up to 5,000 rpm, down to 0, and so on, to configure what controllers affect fan rpm and what sensors are die temperatures, etc. (See below: GPU die temperature in red and fan rpm in light blue:)
![image](https://user-images.githubusercontent.com/64190512/174242404-a859e75d-7b40-44ff-8750-2b841304ecd6.png)
* I use CoreCtrl to configure my GPU's clock and fan curve. AFAIK, there is no problem there; the limit of it's issues is needing to be restarted when I have re-run `pwmconfig` in order for the fan curve to come back into effect: when I run `killall corectrl` and then manually relaunch it, the fans are properly controlled again and return to their normal 500-600 rpm range and the temperature drops down back into the 50s.
![image](https://user-images.githubusercontent.com/64190512/174244014-3f4566f8-e406-4e84-81c9-90e9e76a9b76.png)

If you have any suggestions for any further debug/log information that I should add, please tell me.

**Comment by toydotgame** [comment]
**Update:** Now, instead of 4 rpm, the fans sit in the 530 range. Same issue with the same behaviour, but something's changed to make it sit at this number now.

**Comment by toydotgame** [comment]
**Update 2:** Now averages 2 rpm. No clue.

**Comment by toydotgame** [comment]
Closing issue temporarily as, although my fans get stuck at 0 rpm after unlocking, it goes up to the right number in the 400-500 range (correct rpm according to my fan curve), and it seems to be dynamically responding to changes in GPU die temperature in accordance with my curve as well.
I'll re-open this if the curve stops being listened to, but hopefully I won't have to.

**TL;DR:** Some random commit fixed this issue I think.
